pyproject.toml

```diff
@@ -169,6 +169,7 @@ opentelemetry-sdk = ">=1.25.0"
 truss_transfer="0.0.1"
 uvicorn = ">=0.24.0"
 uvloop = ">=0.17.0"
+websockets = ">=13.0"
```

Comment:
`websockets` 14+ had some incompatibility with truss

Reply:
🤦 good call, I had already installed `13.1` as a compatible version locally which is why `poetry lock` didn't complain here